java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -36,10 +50,16 @@ public class KafkaConsumerConfig {
     private final String saslLoginCallbackClass = "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler";
 
 
-    public KafkaConsumerConfig(String bootstrapServers, String topic, String groupId, String clientRack, Long messageCount,
+    public KafkaConsumerConfig(String serviceName, String tracesExporter, String metricsExporter,boolean openTracingEnabled,  boolean openTelemetryEnabled,
```

Comment:
It's best to keep the most important arguments first, so it would make more sense to add new arguments at the end instead.